guides/v2.1/architecture/archi_perspectives/components/modules/mod_and_areas.md

```diff
@@ -3,7 +3,7 @@ group: architecture-guide
 title: Modules and areas
 menu_title: Modules and areas
 redirect_from:
-  - /guides/v2.0/architecture/modules/mod_and_areas.html
+  - /guides/v2.3/architecture/modules/mod_and_areas.html
```

Comment:
This (and similar in files below) redirects a user form 2.3 link to the 2.1 topic. Is it expected behavior?

Reply:
I guess this file is a source for a symlink in 2.3

Reply:
yes, unfortunately, it's a side effect of symlinks. in symlinked files, whatever entry comes first in the metadata wins.